modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/portlet/ContentDashboardAdminPortlet.java

```diff
@@ -125,7 +126,11 @@ public void render(
 						_http, _language, liferayPortletRequest,
 						liferayPortletResponse, _portal),
 					_itemSelector, liferayPortletRequest,
-					liferayPortletResponse, _portal, searchContainer);
+					liferayPortletResponse, _portal,
+					ResourceBundleUtil.getBundle(
+						"content.Language", _portal.getLocale(renderRequest),
```

Comment:
I think this is the same as ResourceBundleUtil.getBundle(
						_portal.getLocale(renderRequest),getClass())